src/utils/wallet.js

```diff
@@ -316,6 +316,32 @@ const wallet = {
     store.dispatch(partiallyUpdateHistoryAndBalance({ tokensHistory, tokensBalance }));
   },
 
+  /**
+   * Fetches both history and balance for the affected tokens in updatedBalanceMap
+   *
+   * @param {HathorWallet} wallet object
+   * @param {Object} updatedBalanceMap An object containing tokens as the keys and their updated balances as values
+   *
+   * @memberof Wallet
+   * @inner
+   **/
+  async handlePartialUpdate (wallet, updatedBalanceMap) {
+    const tokens = Object.keys(updatedBalanceMap);
+    const tokensHistory = {};
+    const tokensBalance = {};
+
+    for (const token of tokens) {
+      /* eslint-disable no-await-in-loop */
```

Comment:
I suggest disabling this restriction on the whole application.
`await` in loops are very useful!